Cairo/Perspex.Cairo/Media/DrawingContext.cs

```diff
@@ -72,10 +72,11 @@ public void Dispose()
         {
             this.context.Dispose();
 
-            if (this.surface is Cairo.Win32Surface)
-            {
+         //   if (this.surface is Cairo.Win32Surface)
```

Comment:
Why is this commented out? Commented out code tends to get confusing, so might be better if you could remove it, or at least add a comment saying why it's commented out.